action.js

```diff
@@ -67,20 +67,26 @@ const getBodyFromGitHubContext = (team_id, github) => {
     }
   ];
 
+  console.log(get(github, ["context","payload"]))
+  process.exit();
```

Comment:
still a log and exit.

Reply:
Yup, this is intentional. This is the easiest way to see the payload that we need to test / send to API.